mate-api.md

```diff
@@ -450,4 +528,94 @@ if (User.typingSpeedTests?.length > 0) {
   // ...
 }
 
+```
+
+4\. Database
```

Comment:
Wdyt about adding convention for indexes/constraints names? I was thinking about adding this point some time ago. I didn't add this point, cause it's a rather rare case when people name indexes/constraints in a wrong way

Here is the example of such case: https://github.com/mate-academy/website/pull/15822#discussion_r1041430344

And here is an example of some rules for naming constraints/indexes in `PostgreSQL`: https://gist.github.com/popravich/d6816ef1653329fb1745